tmt/steps/provision/vagrant.py

```diff
@@ -514,19 +670,25 @@ def vf_restore(self):
 
 
     ## Helpers ##
-    def info(self, key = '', val = '', color = 'green'):
+    def info(self, key = '', val = '', color = 'blue', level = 1):
```

Comment:
What functionality needs to be added into shared logging functions so that these duplicate implementations would not be necessary?

Reply:
I'm only overloading the method to add additional functionality (therefore passing all that's needed). There's only one functionality I'm avare of that's missing which is [hr()](https://github.com/psss/tmt/pull/294/files/42a975b7150743f49c31d69f365ba74c100a48de#diff-b9519ec24079632bb8030f8a2934cd0eR713).

And possibly some negligible enhancements. I can try to move it to `util.py` if you want. As it's currently overloading just the name = IOW not reimplementing, is it that big an issue?

Reply:
I would just recommend to prevent any unnecessary code duplication.

Reply:
I'll simply try to comment it out, and if it cannot be done, investigate more when I get some spare cycles.